frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTable.tsx

```diff
@@ -55,34 +55,40 @@ const PoliciesTable = ({
 }: IPoliciesTableProps): JSX.Element => {
   const { config } = useContext(AppContext);
 
-  const emptyState = () => {
-    const emptyPolicies: IEmptyTableProps = {
-      graphicName: "empty-policies",
-      header: "You don't have any policies",
-      info:
-        "Add policies to detect device health issues and trigger automations.",
-    };
-    if (canAddOrDeletePolicy) {
-      emptyPolicies.primaryButton = (
-        <Button
-          variant="brand"
-          className={`${baseClass}__select-policy-button`}
-          onClick={onAddPolicyClick}
-        >
-          Add policy
-        </Button>
-      );
-    }
-    if (searchQuery) {
-      delete emptyPolicies.graphicName;
-      delete emptyPolicies.primaryButton;
-      emptyPolicies.header = "No matching policies";
-      emptyPolicies.info = "No policies match the current filters.";
-    }
-
-    return emptyPolicies;
+  const emptyState: IEmptyTableProps = {
+    graphicName: "empty-policies",
+    header: "You don't have any policies",
+    info:
+      "Add policies to detect device health issues and trigger automations.",
   };
 
+  if (currentTeam?.id === null || currentTeam?.id === -1) {
```

Comment:
Good to use [`APP_CONTEXT_ALL_TEAMS_ID`](https://github.com/fleetdm/fleet/blob/79829e28f7eeea925d7f2222fba42d1da50d0300/frontend/interfaces/team.ts#L127) for this kind of thing to a) make the purpose of the condition clear and b) in the unlikely event we ever change the value, to make our lives easier

Reply:
Good tip, done.